containerized_ml_models/api/model.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+import pickle as pkl
+import numpy as np
+
+
+class Model:
+
+    __create_key = object()
```

Comment:
What is create key?

Reply:
It's part of implementation of singleton design pattern. This object is kinda key, which allows to invoke __init__ without getting error. See https://stackoverflow.com/questions/8212053/private-constructor-in-python .

Reply:
But does it create something? You should use verbs in names only if something is doing something. For example __creation_key would be more suiting (or some other name)

Reply:
fixed